climpred/checks.py

```diff
@@ -143,3 +143,21 @@ def match_initialized_vars(init, ref):
             f'got {init_vars} for init and {ref_vars} for ref.'
         )
     return True
+
```

Comment:
Here's how I am checking the datetime/cftime at `esmtools`:

```python
def is_time_index(xobj, kind):
    """
    Checks that xobj coming through is a DatetimeIndex or CFTimeIndex.
    This checks that `esmtools` is converting the DataArray to an index,
    i.e. through .to_index()
    """
    if not (isinstance(xobj, xr.CFTimeIndex) or isinstance(xobj, DatetimeIndex)):
        raise ValueError(
            f'Your {kind} object must be either an xr.CFTimeIndex or '
            f'pd.DatetimeIndex.'
        )
    return True
```

You'll want to do: `xobj['init'].to_index()`. Probably within the prediction module. Also need to do `from pandas.core.indexes.datetimes import DatetimeIndex`.